lib/ASTGen/Sources/ASTGen/Types.swift

```diff
@@ -244,38 +237,25 @@ extension ASTGenVisitor {
     _ elementList: TupleTypeElementListSyntax,
     action: (BridgedArrayRef) -> T
   ) -> T {
-    var elements = [BridgedTupleTypeElement]()
-    for element in elementList {
-      var nameText = element.name?.text
-      let name = nameText?.withBridgedString { bridgedName in
-        return ASTContext_getIdentifier(ctx, bridgedName)
-      } ?? nil
-      let nameLoc = bridgedSourceLoc(for: element.name)
-      var secondNameText = element.secondName?.text
-      let secondName = secondNameText?.withBridgedString { bridgedName in
-        return ASTContext_getIdentifier(ctx, bridgedName)
-      } ?? nil
-      let secondNameLoc = bridgedSourceLoc(for: element.secondName)
-      let colonLoc = bridgedSourceLoc(for: element.colon)
+    return elementList.map { element in
+      let (name, nameLoc) = self.bridgedIdentifierAndSourceLoc(for: element.name)
+      let (secondName, secondNameLoc) = self.bridgedIdentifierAndSourceLoc(for: element.secondName)
       var type = visit(element.type).rawValue
       if let ellipsis = element.ellipsis {
-        let ellipsisLoc = bridgedSourceLoc(at: ellipsis.positionAfterSkippingLeadingTrivia)
-        type = VarargTypeRepr_create(self.ctx, type, ellipsisLoc)
+        type = VarargTypeRepr_create(self.ctx, type, self.bridgedSourceLoc(for: ellipsis))
       }
-      let trailingCommaLoc = bridgedSourceLoc(for: element.trailingComma)
-
-      elements.append(
-        BridgedTupleTypeElement(
-          Name: name,
-          NameLoc: nameLoc,
-          SecondName: secondName,
-          SecondNameLoc: secondNameLoc,
-          UnderscoreLoc: nil, /*N.B. Only important for SIL*/
-          ColonLoc: colonLoc,
-          Type: type,
-          TrailingCommaLoc: trailingCommaLoc))
-    }
-    return elements.withBridgedArrayRef { elements in
+
+      return BridgedTupleTypeElement(
+        Name: name,
+        NameLoc: nameLoc,
+        SecondName: secondName,
+        SecondNameLoc: secondNameLoc,
+        UnderscoreLoc: nil, /*N.B. Only important for SIL*/
+        ColonLoc: self.bridgedSourceLoc(for: element.colon),
+        Type: type,
+        TrailingCommaLoc: self.bridgedSourceLoc(for: element.trailingComma)
```

Comment:
Just me complaining, no action necessary for you in this PR: I know that you didn’t introduce `BridgedTupleTypeElement` but the members of it should really be lowercase so I don’t need to look at uppercase parameter labels in Swift.